modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/list/List.js

```diff
@@ -25,21 +26,31 @@ import FrontendDataSetContext from '../../FrontendDataSetContext';
 import Actions from '../../actions/Actions';
 import ImageRenderer from '../../cell_renderers/ImageRenderer';
 
-const List = ({items, schema}) => {
+const List = ({items, schema, viewsTitle}) => {
 	const {selectedItemsKey} = useContext(FrontendDataSetContext);
 
 	return items?.length ? (
-		<ClayList>
-			{items.map((item, index) => {
-				return (
+		<ClayLayout.Sheet
+			className={classNames('data-views-list-sheet', {
+				'data-views-list': !viewsTitle,
+			})}
```

Comment:
[nit] The naming here seems off. Inheriting from the component name, the main class can be something like `list-sheet`. The conditional class should be something related to header, for example `has-header` or `with-header`.

Reply:
Thank you @markocikos! I've refactored it as in https://github.com/liferay-frontend/liferay-portal/pull/3419#discussion_r1252913943, and changed class names, but I used `no-header` for the conditional class since it's purpose is to override sheet's margin and padding when there's no header. Makes sense?